src/swtpm/swtpm_nvfile.c

```diff
@@ -532,6 +533,17 @@ SWTPM_NVRAM_StoreData_Intern(const unsigned char *data,
             rc = TPM_FAIL;
         }
     }
+    if (rc == 0 && fd >= 0) {
+        TPM_DEBUG("  SWTPM_NVRAM_StoreData: Syncing file %s\n", tmpfile);
+        irc = fsync(fd);
+        if (irc != 0) {
+            logprintf(STDERR_FILENO,
+                      "SWTPM_NVRAM_StoreData: Error (fatal) syncing file\n");
```

Comment:
Would be good to print the _errno_ value for this and the other _fsync()_ calls below.

Reply:
Reasonable. I'll change that.